src/core/ProjectRenderer.cpp

```diff
@@ -242,5 +97,50 @@ void ProjectRenderer::updateConsoleProgress()
 	fflush( stderr );
 }
 
+// gets a buffer and some data as input
```

Comment:
I think saker said on discord about this. The comment should go in the header file as 
```
/*
* First line
* Second line
* ... 
*/
``` 
format

Reply:
Shouldn't it look like this?:

```
/**
* First line
* Second line
* ... 
*/
```

And does this mean that I need to rewrite every comment I wrote in my PRs?

Reply:
Yeah this is what i meant, and regarding re writing, don't rewrite everything, just the ones you use to describe the function.